src/common/actions/repository-input.js

```diff
@@ -0,0 +1,69 @@
+import 'isomorphic-fetch';
+import parseGitHubUrl from 'parse-github-url';
+
+export const CHANGE_REPOSITORY_INPUT = 'CHANGE_REPOSITORY_INPUT';
```

Comment:
There's nothing about the action that's linked to inputs, or change events, or anything at a component level, naming the action with reference to inputs or events needlessly couples it to the component here.